test/helpers/constants.ts

```diff
@@ -2,8 +2,8 @@ import { ethers } from "hardhat";
 import { ICurvePriceConfig } from "../../src/deploy/missions/types";
 
 export const DEFAULT_RESOLVER_TYPE = "address";
-export const ZNS_DOMAIN_TOKEN_NAME = "ZERO ID";
-export const ZNS_DOMAIN_TOKEN_SYMBOL = "ZID";
+export const ZNS_DOMAIN_TOKEN_NAME = "Zero Name Service";
+export const ZNS_DOMAIN_TOKEN_SYMBOL = "ZNS";
```

Comment:
revert this back to what it was

Reply:
It is still being "Zero Name Service" and "ZNS" in my IDE for some reason